README.md

```diff
@@ -216,4 +217,4 @@ This can also be run via screen/tmux or detached like a daemon.
 
 # Configuration
 
-All the model parameters are configurable inside discord. Type `!gp` to view all the configurable parameters, and use `/settings <param> <value>` to set parameters. For example, if I wanted to change the number of images generated by DALL-E by default to 4, I can type the following command in discord: `/settings num_images 4`
+All the model parameters are configurable inside discord. Type `/settings` to view all the configurable parameters, and use `/settings <param> <value>` to set parameters. For example, if I wanted to change the number of images generated by DALL-E by default to 4, I can type the following command in discord: `/settings num_images 4`

```

Comment:
If you don't mind, could you update the README to reflect the new command structure and resolve the conflicts we have? :)

Reply:
It'll be in a good while if I do it, >17 hours, I only have my laptop for some days now and work from 7-17:30 😅

Reply:
Ah no worries at all, I'll do it! Thank you :) Happy work day